src/fswAlgorithms/attGuidance/flybyPoint/_UnitTest/test_flybyPoint.py

```diff
@@ -0,0 +1,242 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+#
+#   Unit Test Script
+#   Module Name:        flybyPoint
+#   Author:             Riccardo Calaon
+#   Creation Date:      April 11, 2023
+#
+
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+from Basilisk import __path__
+from Basilisk.architecture import messaging
+from Basilisk.fswAlgorithms import (flybyPoint)
+from Basilisk.simulation import simpleNav, spacecraft
+from Basilisk.utilities import (SimulationBaseClass, macros, unitTestSupport)
+from Basilisk.utilities import RigidBodyKinematics as rbk
+
+bskPath = __path__[0]
+fileName = os.path.basename(os.path.splitext(__file__)[0])
+
+@pytest.mark.parametrize("accuracy", [1e-12])
```

Comment:
Are there other useful values to parametrize to test the limits of the module? Perhaps ICs that would fail without the future upgrade?

Reply:
Currently the most meaningful case that I can think of is when initial position and velocity vectors relative to the asteroid are parallel, i.e., the spacecraft is on a collision course with the asteroid. It fails in this case because the flight path angle becomes singular. Do you think I should try to make the module robust to this case?

Reply:
See updated unit test